src/viur_cli/update.py

```diff
@@ -98,36 +100,34 @@ def check_req(projects_requirements_path):
             core_requirements = req
             break
 
-    if not core_requirements:
-        echo_error("could now find core package, please update the core to validate the requirements.txt")
-        return
```

Comment:
Could be a shorter change, with less indent. Or not?

```py
    if not core_requirements:
        return []
```

Reply:
i would like to keep the Error Message. The User of the Software should be aware of the missing package, but we should fox the typo.

Reply:
> The User of the Software should be aware of the missing package, but we should fox the typo.

But there's no package missing ...

Reply:
Well, in that case i would also prefer the solution of @sveneberth

Reply:
```suggestion
    errors = []

    if not core_requirements:
        return errors
       
    core_requirements_obj = utils.requirements_to_dict(parse_requirements(core_requirements, session=PipSession()))

    projects_requirements_obj = utils.requirements_to_dict(
        parse_requirements(projects_requirements_path, session=PipSession())
    )

    for package, options in core_requirements_obj.items():
        if package not in projects_requirements_obj:
            errors.append(f"missing package: {package} with version {options['version']}")
            continue
        elif options["version"] != projects_requirements_obj[package]["version"]:
            errors.append(
                f"version mismatch: expected {options['version']} "
                f"got {projects_requirements_obj[package]['version']}: {package}"
            )
            continue
        else:
            # package exists, test hash
            project_hashes = projects_requirements_obj[package]["hashes"]["sha256"]
            core_hashes = options["hashes"]["sha256"]

            if not set(core_hashes).issubset(set(project_hashes)):
                errors.append(f"package hash mismatch: {package}")

        for error in errors:
            echo_error(error)

    return errors
```